censys/base.py

```diff
@@ -194,15 +194,32 @@ def report(self, query, field, buckets=50):
 
 class CensysAPIBaseTests(unittest.TestCase):
 
+    EXPECTED_MY_ACCOUNT_KEYS = set([
+        "login",
+        "first_login",
+        "last_login",
+        "email",
+        "quota",
+    ])
+
+    EXPECTED_QUOTA_KEYS = set([
+        "used",
+        "resets_at",
+        "allowance"
+    ])
+
+
     @classmethod
     def setUpClass(cls):
         cls._api = CensysAPIBase()
 
+    # as mentioned here https://censysio.atlassian.net/browse/DATA-586
+    # this endpoint no longer returns api id / secret 
     def test_my_account(self):
         res = self._api.account()
-        self.assertEqual(res["api_id"], self._api.api_id)
-        self.assertEqual(res["api_secret"], self._api.api_secret)
-
+        self.assertSetEqual(set(res.keys()), self.EXPECTED_MY_ACCOUNT_KEYS) 
+        self.assertTrue("quota" in res.keys())
```

Comment:
do you need this since `quota` is in `EXPECTED_MY_ACCOUNT_KEYS`?

Reply:
I suppose you're right. That's a avoid exception instinct that isn't a huge issue here